pkg/apis/pipeline/v1beta1/pipeline_types.go

```diff
@@ -220,6 +219,28 @@ type PipelineTask struct {
 	Timeout *metav1.Duration `json:"timeout,omitempty"`
 }
 
+// Matrix is used to fan out Tasks in a Pipeline
+type Matrix struct {
+	// Params is a list of parameters used to fan out the pipelineTask
+	// Params takes only `Parameters` of type `"array"`
+	// Each array element is supplied to the `PipelineTask` by substituting `params` of type `"string"` in the underlying `Task`.
+	// The names of the `params` in the `Matrix` must match the names of the `params` in the underlying `Task` that they will be substituting.
+	Params []Param `json:"params,omitempty"`
+
+	// Include is a list of MatrixInclude
+	Include []Include `json:"include,omitempty"`
+}
+
+// Include is used to specify combinations in the matrix
+type Include struct {
```

Comment:
It looks like `include` is introduced in this PR but not used anywhere. I'd suggest keeping the scope of this PR to just move existing matrix params under the `matrix.params` field, and introduce `include` in a later PR (and update the commit message/pr description and title accordingly)

Reply:
ok，I've removed the `include` and just move existing matrix params under the matrix.params field.

commit message/pr description and title are updated